CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout
+  * Update NativeCheckout version from 0.110.0 to 0.112.1
```

Comment:
Do we think it's worth adding a note about the `BraintreePayPalNativeCheckout` now being GA? We will also need to update the public dev docs as they note that is in beta currently.